ansible/playbook_Test_Scripts/Test_Script

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+printf "Enter the URL:  "			#User Input section
+read gitURL
+printf "Retain the VM? [y/n]\n" 
+read vmChoice
+
+
+cd ~/documents/					
```

Comment:
What if that folder doesn't exist? :-)
Also Perhaps start with `adoptopenjdk` in the folder structure so users know what it is.